kitchen-sink-example.yaml

```diff
@@ -1,11 +1,58 @@
 # The file format version
 file_format: "0.1"
 
+# Configure general attribute limits. See also tracer_provider.span_limits, logger_provider.log_record_limits.
+attribute_limits:
+  # Set the max attribute value size.
+  #
+  # Environment variable: OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT
+  attribute_value_length_limit: 4096
+  # Set the max attribute count.
+  #
+  # Environment variable: OTEL_ATTRIBUTE_COUNT_LIMIT
+  attribute_count_limit: 128
+
 # Configure logger provider.
-logger_provider: {}
+logger_provider:
+  # Configure the log record limits. See also attribute_limits.
+  log_record_limits:
```

Comment:
Going to be a stickler and question if this should be `logrecord_limits` or not since the environment variables use `LOGRECORD` and the spec talks about `logrecord` :)